### PR TITLE
Add UIView+Swizzle file

### PR DESCRIPTION
**Pull request by sai-kishore-swaminathan**
UIApplication receives all the events from the app and therefore is notified when you remove your finger from the screen after a responder object is removed from the hierarchy. 

TODO: Have to properly understand how UIApplication dispatches events to UIWindow. [ will do it at the end of the week when I have finished other important tasks ] ✅ 

Video 

https://github.com/bandlab/TouchVisualizer/assets/84300828/3165c454-9b18-4392-ae32-61734bd2b550

### CODE REVIEW

File: TouchVisualizer/UIWindow+Swizzle.swift [hunk 1]
```diff
@@ -8,8 +8,8 @@ import UIKit
 fileprivate var isSwizzled = false
 
 @available(iOS 8.0, *)
-extension UIWindow {
-	
+extension UIApplication {
```
Comment (kolyasev) [inline TouchVisualizer/UIWindow+Swizzle.swift:11]: Let's rename the file as well.

**Review by kolyasev — APPROVED** [review]

File: TouchVisualizer/Visualizer.swift [hunk 2]
```diff
@@ -40,7 +40,7 @@ final public class Visualizer:NSObject {
     
     // MARK: - Helper Functions
     @objc internal func applicationDidBecomeActiveNotification(_ notification: Notification) {
-        UIApplication.shared.keyWindow?.swizzle()
+        UIApplication.shared.swizzle()
```
Comment (NSErfan) [inline TouchVisualizer/Visualizer.swift:43]: Wait from our conversation during Daily: I thought we couldn't access `UIApplication.shared` instance here.

File: TouchVisualizer/UIWindow+Swizzle.swift (hunk 1 above)
Comment (sai-kishore-swaminathan) [reply TouchVisualizer/UIWindow+Swizzle.swift:11]: [UIAppilcation + Swizzle file name change](https://github.com/bandlab/TouchVisualizer/pull/1/commits/da2bbfe5bbb4be8ba01df32dd9d4274092be7c4f)

File: TouchVisualizer/Visualizer.swift (hunk 2 above)
Comment (sai-kishore-swaminathan) [reply TouchVisualizer/Visualizer.swift:43]: Ah it was a mistake,
I was working with an example app and I had removed this method applicationDidBecomeActiveNotification. 
Denis reminded me about using the shared instance here 👍

**Comment by sai-kishore-swaminathan** [comment]
@kolyasev Please do help me merge the changes once review is done because I do not have access 🙈

**Comment by kolyasev** [comment]
@Aranoledur can you please help to merge the changes.

**Review by kolyasev — APPROVED** [review]

**Pull request merged**